addons/sale_loyalty/models/sale_order.py

```diff
@@ -101,7 +101,8 @@ def action_open_reward_wizard(self):
         claimable_rewards = self._get_claimable_rewards()
         if len(claimable_rewards) == 1:
             coupon = next(iter(claimable_rewards))
-            if len(claimable_rewards[coupon]) == 1:
+            rewards = claimable_rewards[coupon]
+            if len(rewards) and not rewards.multi_product:
```

Comment:
Won't this break if there are multiple claimable rewards ?
`len(rewards) = 3 => bool(len(rewards)) = True`  -> `rewards.multi_product => ERROR`

Reply:
It'll... I lost the `== 1` in my reword...
![image](https://github.com/odoo/odoo/assets/69820391/a4d8138b-2767-4613-82c1-a4b19d5f298c)